apps/backend/src/excelexport/excelexport.service.ts

```diff
@@ -53,20 +54,60 @@ export class ExcelexportService {
     }
   }
 
+  public generateWorkbook(title: string): void {
+    this.workbook.title = title;
+    this.workbook.created = new Date(format(new Date(), 'yyyy, MM, dd'));
+  }
+
   public addSheet(name: string): void {
     this.workbook.addWorksheet(name);
   }
 
+  /* eslint-disable  */
+  public addHeader(sheetName: string, style: any): void {
+    const sheet = this.workbook.getWorksheet(sheetName);
+    sheet.insertRow(1, []);
+    const row = sheet.getRow(1);
+    row.addPageBreak();
+    row.getCell('E').value = sheetName;
+    row.getCell('E').style = style;
+    row.height = 20;
+    sheet.getRow(2).addPageBreak();
+  }
+
+  /*eslint-disable @typescript-eslint/no-unused-vars*/
+  public addCellStyle(
```

Comment:
is it adding style to a cell or a row???

Reply:
resolved in report_two renamed to "addRowStyle"